doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgresql_sink.md

```diff
@@ -236,6 +240,7 @@ Coming soon.
 | postgresql\_username | no | postgres | `postgres` is the default username that is created automatically when install |
 | postgresql\_password | no | N/A | Empty value by default (No password is created when install) |
 | attr\_persistence | no | row | <i>row</i> or <i>column</i>. |
+| attr\_native\_types | no | false | <i>false</i> or <i>true</i>. |
```

Comment:
Maybe the description is too "raw". I'd suggest adding an intra-link to the section where you explain this (e.g. `See more detail in [the specific section](#section1.2.6)` or `<a>`, not sure which one will work).